examples/sentencepiece/run_script.sh

```diff
@@ -0,0 +1,5 @@
+if [ ! -d ".data" ]; then
```

Comment:
This is the entry point into running the example, right? do you have a README or instructions to indicate how to use the example?

Reply:
Add a README file in the example folder.